images/linux/scripts/tests/DotnetSDK.Tests.ps1

```diff
@@ -0,0 +1,26 @@
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1"
+
+$dotnetVersions = (Get-ToolsetContent).dotnet.versions
+
+Describe "Dotnet" {
+
+    Context "Default" {
+        It "Default Dotnet SDK is available" {
+            "dotnet --version" | Should -ReturnZeroExitCode
+        }
+    }
+
+    foreach ($version in $dotnetVersions) {
+        Context "Dotnet $version" {
+            $dotnet = @{ dotnetVersion = $version }
+
+            It "SDK $version is available" -TestCases $dotnet {
+                (dotnet --list-sdks | ConvertTo-Json) | Should -Match "$dotnetVersion.[1-9]*"
```

Comment:
I think it is better to parse `dotnet --list-sdks | ConvertTo-Json` and `dotnet --list-runtimes | ConvertTo-Json` once before tests, may be in `BeforeAll` step